fierpy/fier.py

```diff
@@ -223,12 +252,41 @@ def find_fits(reof_ds: xr.Dataset, q_df: xr.DataArray, stack: xr.DataArray, trai
             )
 
             # pack the resulting statistics in dictionary for the loop
-            stats = [temporal_r2,space_r,space_rmse]
+            #stats = [temporal_r2,space_r,space_rmse]
+            stats = [temporal_r2, temporal_r, space_rmse]
             loop_dict = {f"mode{mode}_order{order}_{k}":stats[i] for i,k in enumerate(dict_keys)}
             # merge the loop dictionary with the larger one
             fit_dict = {**fit_dict,**loop_dict}
 
-    return fit_dict
+    #return fit_dict
```

Comment:
@skd862  is there any particular reason why there was a change from returning the resulting dictionary with the fit info to saving only the coefficients? 

I was returning the dict so that we can use it within the downstream processing and save externally if desired. I feel like saving the coefficients ends up being a lot of files and you don't have the stats...

I think if we want to add the option to save the coefficients we can do something like the following:

```python
def find_fits(reof_ds: xr.Dataset, q_df: xr.DataArray, stack: xr.DataArray, train_size: float = 0.7, random_state: int = 0, out_file=None):
...
if out_file is not None:
   # save coefficients to json or netcdf
   return
else:
   return fit_dict
```

Thoughts?